community-members.md

```diff
@@ -122,12 +122,7 @@ The list of active members (both "approvers" and "maintainers") for OpenTelemetr
 
 Repo: [open-telemetry/opentelemetry-php](https://github.com/open-telemetry/opentelemetry-php)
 
-Approvers:
-- [Brain Akins](https://github.com/bakins), Mailchimp
-
-Maintainers:
-- [Bob Strecansky](https://github.com/bobstrecansky), Mailchimp
-- [Dmitry Krokhin](https://github.com/nekufa), basis.company
+The list of active members (both "approvers" and "maintainers") for the OpenTelemetry PHP API and SDK can be found in the [open-telemetry/opentelemetry-php CONTRIBUTING file](https://github.com/open-telemetry/opentelemetry-php/CONTRIBUTING.md).
```

Comment:
I don't think this path is correct, e.g. https://github.com/open-telemetry/opentelemetry-php/blob/master/README.md

Reply:
https://github.com/open-telemetry/opentelemetry-php/pull/109 needs to land first then it will be

Reply:
Yup I believe @lizthegrey is correct here!

Reply:
Does it need to be:
https://github.com/open-telemetry/opentelemetry-php/blob/master/CONTRIBUTING.md and not:
https://github.com/open-telemetry/opentelemetry-php/CONTRIBUTING.md ?

Note the `blob/master`

Reply:
It does, yes I think.  I'll change it.

Reply:
ah right, needs the tree ref.